app/controllers/assessment/autograde.rb

```diff
@@ -228,4 +235,83 @@ def autograde?(submission)
     not (submission.nil? or submission.handinFile().nil?)
   end
 
+  def extend_config_module
```

Comment:
This code is repeated in submissions_controller and assessments_controller but I haven't had the chance to clean it up yet. I think it'll work if we just remove that method from but no time to test.